code/obj/flock/structure/gnesisturret.dm

```diff
@@ -178,6 +177,8 @@
 	icon = 'icons/misc/featherzone.dmi'
 	icon_state = "stunbolt"
 	cost = 10 //how much gnesis you get per-shot
```

Comment:
Looking at this, was this a bug? Was supposed to be 20?

Reply:
nah, lowered it on purpose to balance it a bit.